src/main/java/edu/utdallas/davisbase/representation/DropTableCommandRepresentation.java

```diff
@@ -0,0 +1,37 @@
+package edu.utdallas.davisbase.representation;
+
+public class DropTableCommandRepresentation implements CommandRepresentation {
+  private String command;
+  private String table;
+
+  public DropTableCommandRepresentation(String command,String table) {
+    this.command= command;
+    this.table = table;
+  }
+
+  public String getTable() {
+    return table;
+  }
+
+  public void setTable(String table) {
```

Comment:
Remove the setters to make this object immutable. _(it provides stronger compile-time correctness guarantees, and we don't the setters after construction anyway)_